js/src/get-started-page/get-started-with-video-card/index.scss

```diff
@@ -0,0 +1,126 @@
+.gla-get-started-with-video-card {
+	flex-direction: column;
+
+	// Adjust <CardHeader> imported from @wordpress/components.
+	.components-card__header {
+		display: flex;
+		flex-direction: row;
+		align-items: center;
+		justify-content: flex-end;
+		min-width: 100%;
+		max-height: 64px;
+		padding: $grid-unit-20;
+		margin: 0;
+		text-align: center;
+
+		& > p {
+			font-size: $helptext-font-size;
+			color: $gray-600;
+		}
+
+		@media (max-width: $break-small) {
+			justify-content: center;
+
+			& > p::after {
+				content: " Google";
+			}
+
+			& > img {
+				display: none;
+			}
+		}
+	}
+
+	.motivation-video {
+		min-width: 100%;
+		margin: 0;
+	}
+
+	// Adjust <CardBody> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__body.components-card__body {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		text-align: center;
+		margin: calc(var(--main-gap) * 1.67) calc(var(--main-gap) * 4.67);
+		padding: 0;
+
+		@media (max-width: $break-small) {
+			margin: $grid-unit-30;
+		}
+	}
+
+	// Repeat selector to make it higher priority.
+	& &__title {
+		padding: 0 $grid-unit-30;
+		margin-bottom: $grid-unit-15;
+	}
+
+	// Repeat selector to make it higher priority.
+	& &__description {
+		margin-bottom: $grid-unit-40;
+		padding: 0 $grid-unit-10;
+	}
+
+	// Repeat selector to make it higher priority.
+	& &__button {
```

Comment:
💅 

Some styling details could be considered.
- The padding of `.components-card__header` still has a lower priority than `@wordpress/components`.
- Specify the proper content height to avoid overriding the spacing on the container. This also brings a flexible layout when the content changes or is dynamic.
- Some CSS styles could be omitted.
- Remove some code comments since `& &__` is not a repeated-selector pattern.

I added a code suggestion block because it might be clearer than using a lot of comments. 🙏 
```suggestion
	// Adjust <CardHeader> imported from @wordpress/components.
	// Repeat selector to make it higher priority.
	.components-card__header.components-card__header {
		justify-content: flex-end;
		padding: $grid-unit * 2.5;

		& > p {
			margin: 0 1em;
			line-height: 16px;
			font-size: $helptext-font-size;
			color: $gray-600;
		}

		@media (max-width: $break-small) {
			justify-content: center;
			text-align: center;
			padding: $grid-unit-20;

			& > p::after {
				content: " Google";
			}

			& > img {
				display: none;
			}
		}
	}

	.motivation-video {
		min-width: 100%;
		margin: 0;
	}

	// Adjust <CardBody> imported from @wordpress/components.
	// Repeat selector to make it higher priority.
	.components-card__body.components-card__body {
		display: flex;
		flex-direction: column;
		align-items: center;
		text-align: center;
		margin: calc(var(--main-gap) * 1.67) calc(var(--main-gap) * 4.67);
		padding: 0;

		@media (max-width: $break-small) {
			margin: $grid-unit-30;
		}
	}

	& &__title {
		padding: 0 $grid-unit-30;
		margin-bottom: $grid-unit-15;
	}

	& &__description {
		margin-bottom: $grid-unit-40;
		padding: 0 $grid-unit-10;
	}

	& &__button {
```

Reply:
I missed the other two code comments below, which are also removable.

Reply:
Thanks for the suggested changes @eason9487 , it's much more cleaner now! I've got a question regarding:

> & &__ is not a repeated-selector pattern.

Agree it's different from the repeating form like `className.className`. But using `& &__` actually solved the priority problem to me in this case. I checked the [MDN CSS Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#how_is_specificity_calculated) doc and it seems like `& &__` matches either the rule of

1. Make better use of the CSS cascade, or
2. Use more specific rules. By indicating one or more elements before the element you're selecting, the rule becomes more specific and gets higher priority

So it could have higher priority to override the `Text` component's style from `@wordpress/components`. I'd like to add some comments like `// Use more specific rules to make it higher priority.` to those use `& &__`, what do you think about it?

Reply:
> So it could have higher priority to override the `Text` component's style from `@wordpress/components`. I'd like to add some comments like `// Use more specific rules to make it higher priority.` to those use `& &__`, what do you think about it?

It might seem a bit strange at first glance to see something like `& &__title`, but the purpose of this syntax becomes clear after thinking about the compiled result like `.card .card__title`. And its purpose is exactly the either or both reasons you mentioned. Since these two purposes are very common rules for writing/reading CSS, they usually don't need to be commented on.

But on the other hand, when styling for a component that is entirely under our control, there is no need to override some other styles, so the use of `&__title` is usually sufficient. So even if the reason for these selectors can be deduced here is to raise the priority over `Text`, it would be still fine to leave comments for this concrete reason.